examples/PySDM_examples/Grabowski_and_Pawlowska_2023/settings.py

```diff
@@ -20,6 +20,8 @@ def __init__(
         initial_relative_humidity: float = 0.97,
         displacement: float = 1000 * si.m,
         mass_accommodation_coefficient: float = 0.3,
+        rtol_thd: float = 1e-8,
```

Comment:
let's introduce a global variable used for both default values here (as well as in the test)
this default could also be defined as the default condensation tolerance divided by 100